effect_player.cpp

```diff
@@ -11,6 +11,10 @@ namespace bnb::oep
     /* effect_player::create */
     effect_player_sptr interfaces::effect_player::create(const std::vector<std::string>& path_to_resources, const std::string& client_token)
     {
+        // We need this line to switch the effect_player to OpenGL instead of METAL, which is default for MacOS.
+        // On other platforms this line does no harm.
```

Comment:
maybe it is worth to replace comment stating that this particular example relies on OpenGL, so it should be explicitly requested, rather than describing some SDK platform-specific quirks